etc/profile-m-z/youtube-viewers-common.profile

```diff
@@ -55,5 +60,10 @@ private-dev
 private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.preload,machine-id,mime.types,nsswitch.conf,passwd,pki,pulse,resolv.conf,ssl,X11,xdg
 private-tmp
 
-dbus-user none
+dbus-user filter
+# allow D-Bus communication with firefox for opening links
+dbus-user.talk org.mozilla.Firefox.*
+dbus-user.talk org.mozilla.firefox.*
+dbus-user.talk org.mozilla.firefox_esr.*
```

Comment:
What about
```suggestion
dbus-user.talk org.mozilla.*
```
to cover all (developeredition, ...).

Reply:
Elegant suggestion! It would sure bring in more sanity regarding future maintenance of the FF related profiles. I realize you can't see and/or respond to each and every thing that comes into the Firejail repo, but if you have some time I would appreciate a review of #5360, as this impacts that too. Regardless, and as a personal note, I would like to thank you for all the work you have and hopefully will be doing for Firejail, top notch :+1: .

Reply:
>     dbus-user.talk org.mozilla.*

Wouldn't this also allow talking to something like Thunderbird?

Or to anything else that Mozilla currently (and eventually) publishes besides
Firefox.

Reply:
Yes

Reply:
@rusty-snake [on Sep 7](https://github.com/netblue30/firejail/pull/5352#discussion_r965036075):

> Yes

Isn't that too broad?

Why let a media player talk to an email client?

Reply:
- What does thunderbird expose via D-Bus?
- Why let it speak with an Browser? An Browser in even more sensitive IMHO.

Anyway, that's the details of the function if somebody want to create a call rule (which I'm still not happy to have them in profiles for maintaince reasons).

- Bus: session
- Bus Name: `org.mozilla.<firefox>.<instance>`
- Object Path: `/org/mozilla/firefox/Remote`
- Interface: `org.mozilla.firefox`
- Method: `OpenURL (url: ay) -> ()`

Reply:
Sorry for the late reply, I've been busy working on something else.

Feel free to keep the dbus part as is.

I think I'll just open a dedicated discussion about it later.